### ISSUE TITLE
Compatibility between gitsome and haxor-news

### ISSUE THREAD

**Issue by jashgala** [donnemartin/gitsome#21]
It seems that both these use different versions of the `prompt-toolkit` which does not allow both to coexist simultaneously. 

haxor-news uses 0.52
gitsome uses 0.51

I hope this compatibility issue can be resolved!

(I don't know if this is a valid issue to be posted here. But since they are both awesome tools developed by the same developer, I think this is a good place to post it.)

**Comment by jashgala** [comment]
I know this may sound reckless. I replaced the prompt-toolkit version in gitsome with 0.52 and rebuilt. It fixed this issue. Should I create a pull-request?

**Comment by donnemartin** [comment]
Sure, thanks.  Here's what changed from 0.51 to 0.52:

```
0.52: 2015-09-24
----------------

Fixes:
- Fix in auto suggestion: hide suggestion when accepting input.
```

Can you also verify auto suggestions still work properly?  

https://github.com/donnemartin/gitsome#fish-style-auto-suggestions

**Comment by donnemartin** [comment]
Fixed with ba3d2a89f8dd258f6e8e386813333968ebc59461.
